### PR TITLE
v8 9.7.106.18

### PR DESCRIPTION
**Pull request by XuehaiPan**
Created with `brew bump-formula-pr`.

------

Migrate `v8` to `python@3.10`.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @XuehaiPan!

**Review by iMichka — APPROVED** [review]

**Comment by samford** [comment]
@chenrui333 @iMichka Unless I'm missing something, it appears that the latest V8 version from the stable channel is still 9.6.180.21 according to the [page mentioned in the formula](https://omahaproxy.appspot.com). The 9.7.106.18 version used in this PR is from the beta channel instead, so we may want to revert the version bump part of this PR.

**Comment by iMichka** [comment]
Sorry I missed that. Their versioning scheme is hard to follow .. I opened a PR to fix this: #92054
